helloextend-protection/includes/class-helloextend-protection-orders.php

```
@@ -200,14 +206,58 @@
         // check if shipping protection meta exists
         if ($shipping_protection_quote_id) {
 	        // phpcs:disable WordPress.PHP.DevelopmentFunctions
-	        HelloExtend_Protection_Logger::helloextend_log_notice('Shipping Protection Meta Exists: ' . print_r($shipping_protection_quote_id, true));
+            if ($this->settings['enable_helloextend_debug'] == 1) {
+                HelloExtend_Protection_Logger::helloextend_log_debug('Shipping Protection Meta Exists: ' . print_r($shipping_protection_quote_id, true));
+            }
 	        // phpcs:enable
 
+            // on order completion send shipmentInfo to activate the contract, otherwise send an empty array to create the contract
+            $shipmentInfo = array();
+            $store_raw_country = get_option('woocommerce_default_country');
```

PHPMD (2.15.0)

The variable $store_raw_country is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)